standalone/src/main/resources/conf/wrapper.conf

```diff
@@ -117,6 +117,14 @@ wrapper.java.additional.17=-Dlog4j2.disable.jmx=true
 #wrapper.java.additional.<n>=-XX:GCLogFileSize=1M
 #wrapper.java.additional.<n>=-XX:ErrorFile=%MULE_HOME%/logs/err.log
 
+# FIPS provider settings, defaults to Bouncy Castle if FIPS is enabled and these are not enabled.
```

Comment:
Plese also add this if needed to mule-ee-distributions.

Reply:
this soudln't be here for the CE distro, it is showcasing a feature that is not available.